Tests/js/views/services/assets/ez-contenteditviewservice-tests.js

```diff
@@ -405,8 +405,65 @@ YUI.add('ez-contenteditviewservice-tests', function (Y) {
         },
     });
 
+    getEditViewParametersTest = new Y.Test.Case({
+        name: "eZ Location View View Service getViewParameters tests",
+
+        setUp: function () {
+            this.app = new Y.Mock();
+            this.content = {};
+            this.contentType = {};
+            this.location = {};
+            this.owner = {};
+            this.version = {};
+            this.closeRedirectionUrl = {};
```

Comment:
for this test case, you don't need to inject the complete list of attributes. closeRedirectionUrl, discardRedirectionUrl, publishRedirectionUrl, languageCode are useless.